commands/sort.md

```diff
@@ -103,6 +103,22 @@ It is also possible to `!GET` the element itself using the special pattern `#`:
 SORT mylist BY weight_* GET object_* GET #
 ```
 
+## Restrictions for using external keys
+
+When enabling `Redis cluster-mode` there is no way to guarantee the existence of the external keys on the node which the command is processed on.
+In this case, any use of `GET` or `BY` which reference external key pattern will cause the command to fail with an error.
+
+Starting from Redis 7.0, any use of `GET` or `BY` which reference external key pattern will only be allowed in case the current user running the command has full key read permissions.
+Full key read permissions can be set for the user by, for example, specifying `'%R~*'` or `'~*` with the relevant command access rules. You can check the `ACL SETUSER` command manual for more information on setting ACL access rules.
```

Comment:
nit pick, i'd suggest to break that line, one line per sentence, would make future edits easier to review (smaller diff).